python/sandbox/neurospace/neurospace.py

```diff
@@ -20,21 +20,30 @@ def query(self, query_atom):
                 temperature=0)
         txt = response['choices'][0]['message']['content']
         res = re.findall(r'\{.*?\}', txt)
-        new_bindings_set = BindingsSet()
+        new_bindings_set = BindingsSet.empty()
```

Comment:
Reading this change made stop to think about what the correct behavior for an empty query atom is.  Relates to https://github.com/trueagi-io/hyperon-experimental/issues/281